src/app/components/navbar/navbar.html

```diff
@@ -81,6 +81,29 @@
            [user]="user"
            (onClick)="isMenuCollapsed = true"
         ></customcontrols>
+        <li class="nav-item dropdown" ngbDropdown *ngIf="multiLang">
+          <a class="nav-link" href="javascript:void(0)" ngbDropdownToggle>
+            <i class="fa fa-language"></i>
+              {{ "navbar.lang.title" | translate }}
```

Comment:
Maybe the icon itself is enough? Without the label?
On the right side of the navbar others links are all just icons
The Language label could be a tooltip over the icon

Reply:
yes, it is! I'm not particularly attracted by tooltips, especially if the icon itself is already self-explanatory. I think it is already used for the other icons (eg logout).... then it makes sense

Reply:
We can eventually skip the tooltip, main point was more about the removal of the label to only keep the icon

Reply:
Back to this issue: what about the selected language as label? alternatively, indicate the selected language in the drop-down menu

Reply:
pensavo a qualcosa del genere

![image](https://github.com/rapydo/rapydo-angular/assets/11958518/6b105efa-2c1b-4d93-86a5-7b8d84f6a0e4)